geocoding_platform_interface/lib/src/models/location.dart

```diff
@@ -19,13 +19,13 @@ class Location {
   });
 
   /// The latitude associated with the placemark.
-  final double latitude;
+  final double? latitude;
 
   /// The longitude associated with the placemark.
-  final double longitude;
+  final double? longitude;
 
   /// The UTC timestamp the coordinates have been requested.
-  final DateTime timestamp;
+  final DateTime? timestamp;
```

Comment:
I feel that once we do have a `Location` from the native side it's properties are mandatory and should not be allowed to be `null`.